src/server/JasmineGraphInstanceService.cpp

```diff
@@ -3548,6 +3702,8 @@ map<long, long> calculateOutDegreeDist(string graphID, string partitionID, int s
                                                                      graphDBMapLocalStores,
                                                                      graphDBMapCentralStores);
 
+    instance_logger.log("degreeDistribution size " + to_string(degreeDistribution.size()), "info");
```

Comment:
Degree distribution size should be logged against the graph ID. Otherwise, it is hard to identify for which graph this degree distribution size corresponds to.

Reply:
Fixed

Reply:
Ok